python/tskit/trees.py

```diff
@@ -2370,6 +2370,67 @@ def kc_distance(self, other, lambda_=0.0):
         """
         return self._ll_tree.get_kc_distance(other._ll_tree, lambda_)
 
+    def split_polytomies(
+        self,
+        *,
+        epsilon=None,
+        method=None,
+        record_provenance=True,
+        random_seed=None,
+    ):
+        """
+        Return a new :class:`.Tree` where extra nodes and edges have been inserted
+        so that any any node ``u`` with greater than 2 children --- a multifurcation
+        or "polytomy" --- is resolved into successive bifurcations. New nodes are
+        inserted at times fractionally less than than the time of node ``u``
+        (controlled by the ``epsilon`` parameter).
+
+        If the ``method`` is ``"random"`` (currently the only option, and the default
+        when no method is specified), then for a node with :math:`n` children, the
+        :math:`(2n - 3)!!` possible bifurcating topologies are produced with equal
+        probability.
+
+        .. fixme:: Looks like this function is wrong
```

Comment:
?

Reply:
What's the correct number here? (2n - 3)!! is clearly very wrong indeed, and (2n - 3)! is just wrong.

Reply:
Good point, sorry. Don't know how that snuck in. https://www.jstor.org/stable/2412810?seq=2 : 
<img width="88" alt="Screenshot 2020-11-19 at 23 25 37" src="https://user-images.githubusercontent.com/4699014/99736587-8ea0fc80-2abe-11eb-8f96-5333757c4864.png">

Reply:
Ah, [here](https://oeis.org/A005373/a005373.pdf) it is. We're missing a bunch of terms..

Reply:
I beat you to it :)

Reply:
I think these two expressions are the same, actually, by the conventions of double factorial. https://en.wikipedia.org/wiki/Double_factorial

Reply:
Oh yes. I must have looked that up and forgotten I had done so! Thanks @brianzhang01 . The double factorial thing is nice and concise.